### ISSUE TITLE
打开后图片会重复出现，视频播放卡顿

### ISSUE THREAD

**Issue by exception63** [Eis4TY/XR-Stereoscopic-Viewer#12]
目前测试的机器是 Quest3。通过 sidequest导入个人的 3D 资源后，所有内容都会重复显示（包括自带的那几张图片）。而且测试了一个 200M 的 3d 视频，目前几乎无法播放，卡帧严重。

**Comment by Eis4TY** [comment]
是用的最新版本吗？看看文件夹里的文件有没有重复？大视频确实可能卡顿

**Comment by exception63** [comment]
> 是用的最新版本吗？看看文件夹里的文件有没有重复？大视频确实可能卡顿

看过文件夹，并没有重复的文件。刚刚我退出去再进入应用，问题已经消失。多谢你了，这个 app 很棒。

**Comment by Leegroup0111** [comment]
Version 0.2 has made great progress. I tried to play some videos, but found that there would be lags. The higher the frame rate, the more serious the lags. At the same time, I found that the CPU level of the software is very high. Is the current image algorithm mainly relying on the CPU? Sharing the two videos I used for testing, I hope they will be helpful.
https://drive.google.com/drive/folders/1EyiiFgUiRW3Bmm0ihOuEQmggSiwUiSJC?usp=sharing

**Comment by Eis4TY** [comment]
> Version 0.2 has made great progress. I tried to play some videos, but found that there would be lags. The higher the frame rate, the more serious the lags. At the same time, I found that the CPU level of the software is very high. Is the current image algorithm mainly relying on the CPU? Sharing the two videos I used for testing, I hope they will be helpful.

Yes, I'm so sorry for that. I use the Videoplayer component of Unity, which is very inefficient, and I can't find a better free solution at the moment.

**Comment by lidae1987** [comment]
Ijkplayer、ExoPlayer、VLC，应该都是开源的吧，好似Pico的本地播放器用的是ijk

**Comment by Eis4TY** [comment]
> VLC

好的谢谢，但是涉及到安卓层的东西我就不太懂了😂
